packages/wrangler/src/deploy/index.ts

```diff
@@ -375,7 +367,7 @@ export async function deployHandler(
 		jsxFragment: args.jsxFragment,
 		tsconfig: args.tsconfig,
 		routes: args.routes,
-		experimentalAssets: experimentalAssetsDirectory,
+		experimentalAssets: experimentalAssets?.directory,
```

Comment:
I think this needs to be the `resolvedExperimentalAssetsPath`, right?

Reply:
ah yes, thank you for catching that!